src/modules/components/commentsToList.js

```diff
@@ -0,0 +1,10 @@
+export default function commentsToList(comments) {
```

Comment:
**Changes Required**

- [ ] Thanks for implementing these functionalities. However, I suggest we should prefer the use of `ES6 Arrow functions`. Let us try and refactor these functions using the `ES6 Arrow Functions.`